go/pkg/client/cas.go

```diff
@@ -25,32 +26,122 @@ import (
 	log "github.com/golang/glog"
 )
 
-// UploadIfMissing stores a number of uploadable items.
-// It first queries the CAS to see which items are missing and only uploads those that are.
-// Returns a slice of the missing digests.
-func (c *Client) UploadIfMissing(ctx context.Context, data ...*chunker.Chunker) ([]digest.Digest, error) {
```

Comment:
could you keep interface unchanged?
we rely on missing digest returned by this function.

Reply:
Done!